dist/aws-sdk-core-react-native.js

```diff
@@ -5305,6 +5305,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	 *       failing to establish a connection with the server after
 	 *       `connectTimeout` milliseconds. This timeout has no effect once a socket
 	 *       connection has been established.
+	 *     * **lookup** [Function] &mdash; Custom DNS lookup function.
```

Comment:
All the diffs in `dist/` folder are not needed. They are generated by build script.

Reply:
removed